advanced_alchemy/service/_async.py

```diff
@@ -433,7 +433,7 @@ async def upsert_many(
 
     async def get_or_upsert(
         self,
-        match_fields: list[str] | None = None,
+        match_fields: list[str] | str | None = None,
```

Comment:
Intended?

Reply:
Yep.  This was one of the few places I had not made that update.